include/CXXGraph/Node/Node_decl.h

```diff
@@ -48,6 +48,7 @@ class Node {
   const CXXGraph::id_t &getId() const;
   const std::string &getUserId() const;
   const T &getData() const;
+  T &getData() ;
```

Comment:
Please remove the extra space before the semicolon. And the & should be next to T. Additionally, the function can be marked `const` (however the type returned should not be `const T&`).

Reply:
i.e. `T& getData() const;`